includes/Core/Assets/Assets.php

```diff
@@ -409,14 +409,41 @@ private function get_assets() {
 				'googlesitekit-datastore-site',
 				array(
 					'src'          => $base_url . 'js/googlesitekit-datastore-site.js',
-					'dependencies' => array( 'googlesitekit-api', 'googlesitekit-data' ),
+					'dependencies' => array(
+						'googlesitekit-api',
+						'googlesitekit-data',
+					),
+					'before_print' => function( $handle ) use ( $base_url ) {
+						$current_entity = $this->context->get_current_entity();
+						$inline_data = array(
+							'adminRootURL'        => esc_url_raw( get_admin_url() . 'admin.php' ),
+							'ampMode'             => $this->context->get_amp_mode(),
+							'currentEntityID'     => $current_entity ? $current_entity->ID : null,
+							'currentEntityTitle'  => $current_entity ? $current_entity->title : null,
+							// TODO: This seems like an incomplete approach to the suggestion in
+							// https://github.com/google/site-kit-wp/issues/1000
+							// It should possibly return `''` or `false`.
+							'currentEntityType'   => is_home() ? 'home' : 'post',
```

Comment:
I wasn't clear about the approach this value should take, so if you could clarify what's needed here that'd be awesome. Thanks! 😄